src/plugins/image_display/ImageDisplay.cc

```diff
@@ -15,16 +15,21 @@
  *
 */
 
-#include <QQuickImageProvider>
+#include "ImageDisplay.hh"
+
+#include <algorithm>
 #include <iostream>
+#include <string>
+#include <vector>
+
+#include <QQuickImageProvider>
 
 #include <ignition/common/Console.hh>
 #include <ignition/common/Image.hh>
 #include <ignition/plugin/Register.hh>
 #include <ignition/transport/Node.hh>
 
 #include "ignition/gui/Application.hh"
-#include "ImageDisplay.hh"
```

Comment:
This change makes the local code checker fail:

```
./src/plugins/image_display/ImageDisplay.cc:25:  Found C system header after C++ system header. Should be: ImageDisplay.h, c system, c++ system, other.  [build/include_order] [4]
```

Reply:
I believe the error is actually coming from `#include <QQuickImageProvider>` (it looks like the local checker interprets this as a C system header). I'm working on a fix.

Reply:
Should be fixed now in https://github.com/ignitionrobotics/ign-gui/pull/144/commits/3b1494cf12ae07bc1f53429fca8cc1ccd48e4ce1.